recipe/meta.yaml

```diff
@@ -21,9 +21,10 @@ build:
   skip: True
   {% endif %}
   script:
-    - cp {{ os.environ["FEEDSTOCK_ROOT"] }}/LICENSE.txt ${RECIPE_DIR}/LICENSE.txt            # [linux]
-    - cp ${RECIPE_DIR}/../LICENSE.txt ${RECIPE_DIR}/LICENSE.txt                              # [osx]
-    - COPY "%RECIPE_DIR%\\..\\LICENSE.txt" "%RECIPE_DIR%\\LICENSE.txt"                       # [win]
+    # Copy feedstock license file to recipe directory
+    - pushd {{ RECIPE_DIR }}
+    - cp recipe-scripts-license.txt LICENSE.txt
```

Comment:
Doesn't this rely on an implementation detail in smithy? It is also very opaque since this file appears nowhere in the actual recipe.

Reply:
Am ok going back to the previous approach. Was just trying this to see if we could simplify

Reply:
yeah we should go back and add a `.get`